studiopress/genesis-custom-blocks
# Clean up template editor for release

## Changes
* Changes the template notice, as the recommended approach is the new editor:
>Edit the template in the Template Editor or add this template file to your theme:


![template-editor-here](https://user-images.githubusercontent.com/4063887/118071515-8a6ae300-b36d-11eb-9ac5-fd1f6571901b.png)
* The message when there is no template found mentions the Template Editor first:
>No Template Editor markup or template file was found: blocks/here-is-something/block.php

![template-editor](https://user-images.githubusercontent.com/4063887/118071726-f0f00100-b36d-11eb-8f45-f7820680baf6.png)

* Adds more notices, like if you need to edit the block to preview it

## Testing instructions
1. `composer install && npm install && npm run dev`
2. `/wp-admin` > Custom Blocks > Add New
3. Add a field: 
![wiht-sdf](https://user-images.githubusercontent.com/4063887/118789794-c06f0200-b85a-11eb-8405-f140320a29a6.png)
4. Click 'Front-end Preview': 
5. Click 'Editor Preview':
![ijodsaf-sdaf](https://user-images.githubusercontent.com/4063887/118790022-fca26280-b85a-11eb-8665-40469dd602f7.png)
6. Expected: That changes the display to the editor preview: 
![changes-here](https://user-images.githubusercontent.com/4063887/118790647-979b3c80-b85b-11eb-81f5-884070a33fe9.png)
